src/homepageExperience/components/steps/python/InstallDependenciesSql.tsx

```diff
@@ -0,0 +1,44 @@
+import React, {PureComponent} from 'react'
+import CodeSnippet from 'src/shared/components/CodeSnippet'
+import {SafeBlankLink} from 'src/utils/SafeBlankLink'
+
+import {event} from 'src/cloud/utils/reporting'
+
```

Comment:
Is there a reason we're using the class syntax here instead of a functional component with ES6 syntax, or is that just a Ctrl+C relic?

Reply:
relic